lib/engine/game/base.rb

```diff
@@ -1377,12 +1377,12 @@ def compute_stops(route, train = nil)
           stops, revenue = visits.combination(num_stops.to_i).map do |stops|
             # Make sure this set of stops is legal
             # 1) At least one stop must have a token (if enabled)
-            next if train.requires_token && stops.none? { |stop| stop.tokened_by?(route.corporation) }
+            next if train.requires_token && stops.none? { |stop| stop&.tokened_by?(route.corporation) }
 
```

Comment:
When starting a route from validvastock, stop is nil and this line throws an exception. No idea why it's nil

Reply:
Can you write a bug for this and reference that this is a workaround?

Reply:
found the issue. see bellow

Reply:
not changing any base classes